src/pages/Validator/DetailCard.tsx

```diff
@@ -0,0 +1,114 @@
+import {Box, Skeleton} from "@mui/material";
+import * as React from "react";
+import HashButton from "../../components/HashButton";
+import ContentBox from "../../components/IndividualPageContent/ContentBox";
+import ContentRow from "../../components/IndividualPageContent/ContentRow";
+import {HashType} from "../../components/HashButton";
+import RewardsPerformanceTooltip from "../Validators/Components/RewardsPerformanceTooltip";
+import LastEpochPerformanceTooltip from "../Validators/Components/LastEpochPerformanceTooltip";
+import {prettifyTimestamp} from "../utils";
+import {Types} from "aptos";
+import {
+  useGetMainnetValidators,
+  useGetValidatorToOperator,
+} from "../../api/hooks/useGetValidatorSet";
+import {useGetAccountResource} from "../../api/hooks/useGetAccountResource";
+import {ValidatorCardDetailName, validatorCardDetailType} from "./Constants";
+import {useEffect, useState} from "react";
+
+type ValidatorDetailProps = {
+  address: Types.Address;
+};
+
+export default function ValidatorDetailCard({address}: ValidatorDetailProps) {
+  // make sure that addresses will always start with "0x"
+  const addressHex = address.startsWith("0x") ? address : "0x" + address;
+  const {validators} = useGetMainnetValidators();
+  const {isLoading, accountResource} = useGetAccountResource(
+    addressHex,
+    "0x1::stake::StakePool",
+  );
+  const validatorToOperator = useGetValidatorToOperator();
+  const validator = validators.find(
+    (validator) => validator.address === addressHex,
+  );
+  const sx = {display: "flex", justifyContent: "space-between"};
+  const [isSkeletonLoading, setIsSkeletonLoading] = useState<boolean>(true);
+
+  const lockedUntilSecs = accountResource
+    ? BigInt((accountResource.data as any).locked_until_secs)
+    : null;
+  const operatorAddr = validatorToOperator
+    ? validatorToOperator[addressHex]
+    : null;
+  const rewardGrowth = validator ? validator.rewards_growth : null;
```

Comment:
I think you simplify it and do

```suggestion
  const rewardGrowth = validator?.rewards_growth
```